### ISSUE TITLE
Logback logging configuration issue

### ISSUE THREAD

**Issue by albilu** [bucket4j/bucket4j#494]
Is it necessary to have this file [bucket4j-redis/bucket4j-lettuce/src/main/resources/logback-test.xml](https://github.com/bucket4j/bucket4j/blob/master/bucket4j-redis/bucket4j-lettuce/src/main/resources/logback-test.xml) ?

It overides the default Spring boot application logging configuration see logs attached

```
15:24:03,584 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback-test.xml] at [jar:file:/C:/Users/xxx/.m2/repository/com/bucket4j/bucket4j_jdk17-lettuce/8.12.1/bucket4j_jdk17-lettuce-8.12.1.jar!/logback-test.xml]
```

We will like to keep our logging configuration how it is.

Is there any worakaround for this. Thanks

[logs.txt](https://github.com/bucket4j/bucket4j/files/15434984/logs.txt)

**Comment by vladimir-bukhtoyarov** [comment]
It is copy-paste related mistake. I will remove this config file in next release. ETA for new release is not defined, #494 does not look as issue that requires dedicated release, so if problem is serrious for you, then you can temporary remove dependency  from `bucket4j-letucce` and just copy 3 classes directly to your project from [this package](https://github.com/bucket4j/bucket4j/tree/master/bucket4j-redis/bucket4j-lettuce/src/main/java/io/github/bucket4j/redis/lettuce) and [its subpackage](https://github.com/bucket4j/bucket4j/tree/master/bucket4j-redis/bucket4j-lettuce/src/main/java/io/github/bucket4j/redis/lettuce/cas).

**Comment by albilu** [comment]
> It is copy-paste related mistake. I will remove this config file in next release. ETA for new release is not defined, #494 does not look as issue that requires dedicated release, so if problem is serrious for you, then you can temporary remove dependency from `bucket4j-letucce` and just copy 3 classes directly to your project from [this package](https://github.com/bucket4j/bucket4j/tree/master/bucket4j-redis/bucket4j-lettuce/src/main/java/io/github/bucket4j/redis/lettuce) and [its subpackage](https://github.com/bucket4j/bucket4j/tree/master/bucket4j-redis/bucket4j-lettuce/src/main/java/io/github/bucket4j/redis/lettuce/cas).

Works for me copying the files! Thanks for the work

**Comment by martinufo** [comment]
As a workaround, I passed a VM argument to the app launcher: `-Dlogback.configurationFile=logback.xml`. It forces loading `logback.xml` from classpath and skips `logback-test.xml`.

**Comment by ddrzymala** [comment]
I'm also having same issue. If we can latest version asap that would be great. Thanks

**Comment by ddrzymala** [comment]
If you use springboot - you need to use -Dlogging.config=logback.xml instead of -Dlogback.configurationFile=logback.xml - it may be useful before permanent fix is released.
If file is in resource folder you can use -Dlogging.config=classpath:logback.xml as well.

**Comment by vladimir-bukhtoyarov** [comment]
Will be released this weekend with version `8.13.0`
